### PR TITLE
docs: add zoom functionality on images

### LINKED ISSUE #2393: Docs: Add Zoom functionality on images using docusaurus-plugin-image-zoom plugin
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/medusajs/medusa/issues

### Issue Summary

We want to add the ability to zoom into images. We can use the docusaurus plugin [docusaurus-plugin-image-zoom plugin](https://gabrielcsapo.github.io/docusaurus-plugin-image-zoom/docs/getting-started/) to do that.

### How can this issue be resolved?

1. Install and configure [docusaurus-plugin-image-zoom plugin](https://gabrielcsapo.github.io/docusaurus-plugin-image-zoom/docs/getting-started/)

### Are you interested in working on this issue?

- [ ] I would like to fix this issue

### PR DESCRIPTION
**Pull request by aniketh-varma**
Closes #2393 

Added zoom functionality on images in the documentation page using the docusaurus-plugin-image-zoom plugin.

The changes will install the docusaurus-plugin-image-zoom plugin and configure the same.

I have tested the changes locally and did not encounter anything breaking.

Please feel free to suggest any changes. Thanks!

### CODE REVIEW

File: www/docs/docusaurus.config.js [hunk 1]
```diff
@@ -54,6 +55,19 @@ const config = {
       theme: require("prism-react-renderer/themes/vsDark"),
       darkTheme: require("prism-react-renderer/themes/vsDark"),
     },
+    zoom: {
+      selector: '.markdown img',
+      background: {
+        light: 'rgb(255, 255, 255)',
+        dark: 'rgb(50, 50, 50)'
+      },
```
Comment (shahednasser) [inline www/docs/docusaurus.config.js:63]: According to the documentation this should be inside the `config` object

File: www/docs/docusaurus.config.js [hunk 2]
```diff
@@ -54,6 +55,19 @@ const config = {
...
+      config: {
+        container: {
+          left: 250,
+          right: 40
+        }
```
Comment (shahednasser) [inline www/docs/docusaurus.config.js:68]: Why is this necessary? I removed it and nothing changed

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thanks for your contribution! In addition to the comments I added below, can you change the `z-index` of `.medium-zoom-overlay` and `.medium-zoom-image--opened` to `z-index: var(--ifm-z-index-overlay);`? You should be able to add it in `src/css/custom.css`. That way the sidebar doesn't appear when the pic is zoomed.

**Comment by aniketh-varma** [comment]
> Thanks for your contribution! In addition to the comments I added below, can you change the `z-index` of `.medium-zoom-overlay` and `.medium-zoom-image--opened` to `z-index: var(--ifm-z-index-overlay);`? You should be able to add it in `src/css/custom.css`. That way the sidebar doesn't appear when the pic is zoomed.

Sure! I will make the changes

File: www/docs/docusaurus.config.js (hunk 1 above)
Comment (aniketh-varma) [reply www/docs/docusaurus.config.js:63]: Yes, the documentation is not updated. I had tried these `background` properties inside the `config` object and no changes were being reflected.
This is the thread which I referred: https://github.com/gabrielcsapo/docusaurus-plugin-image-zoom/issues/10

File: www/docs/docusaurus.config.js (hunk 2 above)
Comment (aniketh-varma) [reply www/docs/docusaurus.config.js:68]: I had done this to zoom in the image inside the text container and not overflow over the sidebar, but I will make the appropriate changes in the z-index

**Comment by aniketh-varma** [comment]
Hi @shahednasser, I have done the suggested changes. Please have a look.

**Review by shahednasser — APPROVED** [review]

**Comment by shahednasser** [comment]
@aniketh-varma there seems to be an issue with `yarn.lock`. Can you try removing `node_modules` in `www/docs`, running `yarn install`, and pushing any changes?

**Comment by aniketh-varma** [comment]
> @aniketh-varma there seems to be an issue with `yarn.lock`. Can you try removing `node_modules` in `www/docs`, running `yarn install`, and pushing any changes?

Sure! I will do the same.

**Comment by shahednasser** [comment]
Awesome thanks a lot! 🙏🏻 @aniketh-varma
